tests/unit/kml_documents/expected_enumerated_positions.kml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:gx="http://www.google.com/kml/ext/2.2">
+    <Document id="25">
```

Comment:
Just curious on how these ids come about? Why is it 25

Reply:
Did some investigation and it's caused by a global class variable. Added a workaround.